atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -25,11 +25,11 @@
 import com.palantir.atlasdb.keyvalue.api.RowResult;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
 import com.palantir.atlasdb.spi.KeyValueServiceConfig;
-import com.palantir.atlasdb.transaction.api.annotations.ReviewedRestrictedApiUsage;
 import com.palantir.atlasdb.transaction.api.exceptions.MoreCellsPresentThanExpectedException;
 import com.palantir.atlasdb.transaction.service.TransactionService;
 import com.palantir.common.annotation.Idempotent;
 import com.palantir.common.base.BatchingVisitable;
+import com.palantir.lock.annotations.ReviewedRestrictedApiUsage;
```

Comment:
Should this be in some shared/utils package? How does this annotation actually work? Does this trigger some errorprone thing?

Reply:
It does, you need to add it in the method using the restricted api - see https://github.com/palantir/atlasdb/pull/7305/files#diff-3939f806d02720d19da8a6fd016bab16268d8b80b5926d39840cd799acffadc6R1891

Reply:
> Should this be in some shared/utils package?

Think it should be in `atlasdb-commons` - `com.palantir.common.annotations` seems like a good one. But I'll not do this for now as it seems a bit more involved (would probably extract things out of `atlasdb-common` to `atlasdb-common-api` tbh to achieve that.

Reply:
Where's the bit that actually ties this specific annotation to errorprone failing?